seqr/utils/elasticsearch/es_utils_2_3_tests.py

```diff
@@ -831,6 +832,7 @@ def mock_hits(hits, increment_sort=False, include_matched_queries=True, sort=Non
 
 
 def create_mock_response(search, index=INDEX_NAME):
+    index = safe_redis_get_json('index_alias__{}'.format(index)) or index
```

Comment:
This is hard to read and confused me. Just create some sort of helper dictionary explicitly mapping the expected alias to the expected index and use it here
```
# somewhere at the top of the test file with the other constants
INDEX_ALIAS = '236a15db29fc23707a0ec5817ca78b5e'
ALIAS_MAP = {INDEX_ALIAS: INDEX_NAME}
...
# In create_mock_response
index = ALIAS_MAP.get(index, index)
```

Reply:
Okay. But it seems overkilled to build an alias mapping factory for only one test. I have updated the alias search test by providing the expected index names to `create_mock_response()` function.

Reply:
It is not overkill, its making a static test fixture actually static instead of relying on a helper function that could have a signature change, could be redone, could stop being used entirely, etc. This test is independent of caching and shouldn't start failing because our caching changes, the way you have it implemented introduces an unnecessary dependency